content/docs/0.16.x/manage/shipyard/index.md

```diff
@@ -456,211 +269,3 @@ spec:
 
 **Result:** After extending the shipyard as shown above, remediations should be executed when a problem event is retrieved (see [remediation workflow](../../automated_operations/remediation)).
 
-### Define a trigger for a sequence 
-
-An advanced and powerful feature of the shipyard is that you can define *triggers* to kick-off a sequence. Therefore, a sequence offers the `triggeredOn` property where a list of events can be specified. The event types you can list there are events that refer to the status of a sequence execution. Their name follows the pattern:
```

Comment:
By default Keptn sequences are standalone. They run and they finish. An advanced and powerful feature of the shipyard is that you can *link* sequences together to form chains of sequences. This can be achieved using the `triggeredOn` property....

Reply:
Reworded in reference page

Reply:
I also modified the "Triggers" page to begin with a list of  ways a sequence can be triggered, then put the content we have into a section entitled "Using triggeredOn in a sequence".  We need to go back and add content for the other methods and probably have other work to do but this is a start...